internal/tpm/client_test.go

```diff
@@ -1360,12 +1359,6 @@ func TestSessionGenerateChallenge(t *testing.T) {
 
 			require.NoError(err)
 
-			// we need extra memory in the sim and aren't using the LDevID for this test.
```

Comment:
For posterity, can we record in this PR comment thread why this is going away? I think due to you flushing the SRK [here](https://github.com/flightctl/flightctl/pull/1664/files#diff-c55fdc7dfeb4c0c9984bc3115ad694a781df910120c6b686af3b9b82fb3461afR70) IIRC

Reply:
That is correct. The amount of transient space allocated on TPMs is relatively limited. The resource managers do a decent job of bringing items in and out of memory per connection, but on a single connection we are relatively limited. In our specific case, we were loading the SRK, LAK, and LDevID on startup and never flushing. During the ActivateCredential flow we need to load the EK and, because we were using all available slots, it would fail to load. To get around it, initialization of the client now flushes the SRK after it has loaded the LAK and LDevID. Since we have this extra space, these tests no longer need to flush random handles.